### PR TITLE
LRQA-55984 | master

### PR DESCRIPTION
**Pull request by calvinctvo**
https://issues.liferay.com/browse/LRQA-55984

### CODE REVIEW

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by calvinctvo** [comment]
ci:test

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by calvinctvo** [comment]
ci:test

**Comment by john-co** [comment]
@calvinctvo, there's some test formatting issues. You can check locally with `ant -f build-test.xml run-poshi-validation`

Also please include ticket prefix on the last commit

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro [hunk 1]
```diff
@@ -218,6 +218,32 @@ definition {
 		Page.uploadFileForLogo(logoFile = "${logoFile}");
 	}
 
+	macro changeSinglePageTheme {
+		Click(locator1 = "ControlMenu#EDIT");
+		Pause(locator1 = "5000");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:223]: We should avoid implicit pauses.

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro [hunk 2]
```diff
@@ -218,6 +218,32 @@ definition {
...
+		Click.clickNoMouseOver(
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:224]: Needs spaces between the actions

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro [hunk 3]
```diff
@@ -262,14 +288,6 @@ definition {
 		Uncheck.uncheckHiddenToggleSwitch(locator1 = "ToggleSwitch#INHERIT_MOBILE_DEVICE_RULES_FROM_PAGE");
 	}
 
-	macro editFriendlyURL {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:265]: This is the cause of the poshi validation error. Why do we need to remove this?

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/controlmenu/ControlMenu.path [hunk 4]
```diff
@@ -13,6 +13,11 @@
 	<td>//*[@data-qa-id='controlMenu']//*[@data-qa-id='add']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>EDIT</td>
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/controlmenu/ControlMenu.path:17]: EDIT_PENCIL

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro (hunk 1 above)
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:223]: We have a macro for this already, ContentPages.clickPencil(); It has a implicit pause added...

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by calvinctvo** [comment]
ci:test

**Comment by john-co** [comment]
@calvinctvo, you should run ci:test:relevant next time. ci:test runs all acceptance tests and takes up a lot of resources. It's overkill for adding one testcase...the added test didn't even run in the suite b/c it's not an acceptance test.

**Comment by john-co** [comment]
ci:test:relevant

**Comment by calvinctvo** [comment]
ci:test:nocompile

**Comment by john-co** [comment]
ci:test:nocompile

**Comment by calvinctvo** [comment]
ci:test:relevant

**Comment by calvinctvo** [comment]
ci:test:relevant

**Comment by john-co** [comment]
@calvinctvo there are relevant tests failing. Please take a look and see if related.

**Comment by calvinctvo** [comment]
ci:test:relevant

**Comment by john-co** [comment]
I think the cucumber error is unrelated...not sure what to make of it right now...will rerun tests since infra channel says a similar cucumber error seems random :crossed_fingers:

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**